src/px-script/sidebar/index.js

```diff
@@ -0,0 +1,122 @@
+class Sidebar {
+    constructor (el) {
+        this._el = el;
+        this.sidebarOpen = el.classList.contains("sidebar-open");
+        this.childCount = [...this._el.querySelectorAll(".submenu")].length;
+        this.submenus = this._el.querySelectorAll(".submenu");
+        this.listItems = [...this._el.querySelectorAll("li")].length;
+        this.resizeEventMenuOpen;
+        this.resizeEventSubmenuOpen;
+        this.submenuClosed;
+
+        document.addEventListener("click", e => {
+            if (!e.target.closest(".sidebar") && this.sidebarOpen) {
+                this.closeSidebar();
+                this.hideItems();
+            } else if (!e.target.closest(".submenu") && !this.submenuClosed) {
+                this.submenuCloseAll();
+            }
+        });
+
+        if (this.listItems > 5 || this.childCount > 0) {
```

Comment:
No need to check specifically if it is bigger than `0`

```suggestion
        if (this.listItems > 5 || this.childCount) {
```